Tests/swift-sdk-swift-tests/LocalStorageTests.swift

```diff
@@ -92,4 +92,20 @@ class LocalStorageTests: XCTestCase {
         let fromLocalStorage2:[AnyHashable : Any]? = localStorage.payload
         XCTAssertNil(fromLocalStorage2)
     }
+    
+    func testDeviceId() {
+        let mockDateProvider = MockDateProvider()
+        var localStorage: LocalStorageProtocol = UserDefaultsLocalStorage(dateProvider: mockDateProvider)
+        let deviceId = UUID().uuidString.replacingOccurrences(of: "-", with: "")
```

Comment:
This will not fail because what we are testing here is only saving/retrieval of the value. We could have set any valid string here.  This is just testing whether the value set is the value saved in the storage.

Reply:
But I think you are right. Removing 'replacingOccurrences' is less confusing and more standard so I will remove it.